### PR TITLE
Rule method_length: Add Check for Empty Method

### LINKED ISSUE #488: New rule: no empty methods
I think this rule about empty teardown can be extended to the other methods as well:

https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#dont-add-a-teardown-unless-you-really-need-it

### PR DESCRIPTION
**Pull request by g-back**
- closes #488

### CODE REVIEW

File: src/rules/method_length.ts [hunk 1]
```diff
@@ -36,13 +52,20 @@ export class MethodLength implements IRule {
     const stats = MethodLengthStats.run(obj);
 
     for (const s of stats) {
+      if (s.count === 0 && this.conf.checkEmptyMethods) {
```
[the remark this answers is not in the document]
Comment (g-back) [reply src/rules/method_length.ts]: Good point, still new to TS so I didn't even know there was a difference!

[the remark this answers is not in the document]
Comment (larshp) [reply src/rules/method_length.ts]: I'm not sure if there is in this case, but I've had trouble with this case

File: src/rules/method_length.ts [hunk 2]
```diff
@@ -5,10 +5,16 @@ import {MethodLengthStats} from "../abap/method_length_stats";
 import {IRule} from "./_irule";
 import {BasicRuleConfig} from "./_basic_rule_config";
 
-/** Chcecks that methods do not exceed the set number of statements */
+/** Checks relating to method length. */
 export class MethodLengthConf extends BasicRuleConfig {
   /** Maximum method length in statements */
   public statements: number = 100;
+  public checkEmptyMethods: boolean = true;
```
Comment (larshp) [inline src/rules/method_length.ts]: please add description `/** foobar */`, and suggest renaming to `errorWhenEmpty` or something similar

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**